Makefile.dryice.js

```diff
@@ -447,13 +449,113 @@ function buildAce(options, callback) {
             sanityCheck(options, callback);
         if (options.noconflict && !options.compress)
             buildTypes();
+        extractCss(options);
             
         if (callback) 
             return callback();
         console.log("Finished building " + getTargetDir(options));
     }
 }
 
+function extractCss(options) {
```

Comment:
Is that ok that we don't await on creation of theme files here, and on `extractCss` in general? Feels like `extractCss` should accept a callback to, but I don't really have much context here.